umh-core/docs/production/deployment/docker-compose/configs/nginx.conf

```diff
@@ -0,0 +1,126 @@
+# ==============================================================================
```

Comment:
maybe split this PR up into:
1. one jut for docker compose with nginx (new pattern 1: docker-compose, new pattern 2: nginx for SSL/TLS)
2. one for historian docker compose on top